pandas/core/arrays/timedeltas.py

```diff
@@ -220,9 +264,37 @@ def _generate_range(cls, start, end, periods, freq, closed=None):
 
         return cls._simple_new(index, freq=freq)
 
+    # -----------------------------------------------------------------
+    # DatetimeLike Interface
+    def _unbox_scalar(self, value):
+        if not isinstance(value, (self._scalar_type, type(NaT))):
+            raise ValueError("'value' should be a a Timestamp..")
```

Comment:
Timestamp->Timedelta

Reply:
why does the DatetimeArray version check for isna(value) before the next line but this doesn't?  Other than that, the methods are idenitcal